[2024-12-11T14:52:32.972+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-11T14:52:32.981+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: la_otra_schedule_scraper.merge_all_xml scheduled__2024-12-09T23:30:00+00:00 [queued]>
[2024-12-11T14:52:32.987+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: la_otra_schedule_scraper.merge_all_xml scheduled__2024-12-09T23:30:00+00:00 [queued]>
[2024-12-11T14:52:32.987+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 4
[2024-12-11T14:52:32.994+0000] {taskinstance.py:2889} INFO - Executing <Task(_PythonDecoratedOperator): merge_all_xml> on 2024-12-09 23:30:00+00:00
[2024-12-11T14:52:32.998+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=145) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-11T14:52:33.000+0000] {standard_task_runner.py:72} INFO - Started process 151 to run task
[2024-12-11T14:52:33.000+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'la_otra_schedule_scraper', 'merge_all_xml', 'scheduled__2024-12-09T23:30:00+00:00', '--job-id', '2284', '--raw', '--subdir', 'DAGS_FOLDER/dag_la_otra.py', '--cfg-path', '/tmp/tmpa4875f6i']
[2024-12-11T14:52:33.001+0000] {standard_task_runner.py:105} INFO - Job 2284: Subtask merge_all_xml
[2024-12-11T14:52:33.025+0000] {task_command.py:467} INFO - Running <TaskInstance: la_otra_schedule_scraper.merge_all_xml scheduled__2024-12-09T23:30:00+00:00 [running]> on host b605548c7b8b
[2024-12-11T14:52:33.069+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='la_otra_schedule_scraper' AIRFLOW_CTX_TASK_ID='merge_all_xml' AIRFLOW_CTX_EXECUTION_DATE='2024-12-09T23:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-09T23:30:00+00:00'
[2024-12-11T14:52:33.070+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-11T14:52:33.181+0000] {logging_mixin.py:190} INFO - [INFO] Generado el archivo "all_in_one.xml"
[2024-12-11T14:52:33.182+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-12-11T14:52:33.191+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-11T14:52:33.191+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=la_otra_schedule_scraper, task_id=merge_all_xml, run_id=scheduled__2024-12-09T23:30:00+00:00, execution_date=20241209T233000, start_date=20241211T145232, end_date=20241211T145233
[2024-12-11T14:52:33.214+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-11T14:52:33.224+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-11T14:52:33.239+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
